library/std/src/sys/pal/unix/process/process_unix.rs

```diff
@@ -335,7 +335,7 @@ impl Command {
                 cvt(libc::setuid(u as uid_t))?;
             }
         }
-        if let Some(ref cwd) = *self.get_cwd() {
+        if let Some(ref cwd) = self.get_cwd() {
```

Comment:
I think this no longer needs `ref`.
```suggestion
        if let Some(cwd) = self.get_cwd() {
```

There's another call like this in `process_vxworks.rs` too.